src/libs/SidebarUtils.ts

```diff
@@ -117,12 +117,22 @@ function getOrderedReportIDs(
     betas: Beta[],
     policies: Record<string, Policy>,
     priorityMode: ValueOf<typeof CONST.PRIORITY_MODE>,
-    allReportActions: OnyxCollection<ReportAction[]>,
+    allReportActions: OnyxCollection<ReportActions>,
+    transactionViolations: TransactionViolations,
 ): string[] {
+    const countActions = (actions: ReportActions | null): number => (actions ? Object.keys(actions).length : 0);
+
+    const getReportActionCount = (reportIDKey: string): number | null => {
+        if (!allReportActions?.[reportIDKey]) {
+            return null;
+        }
+        return countActions(allReportActions[reportIDKey]);
```

Comment:
```suggestion
    const getReportActionCount = (reportIDKey: string): number => {
        if (allReportActions?.[reportIDKey]) {
            return Object.keys(allReportActions[reportIDKey]).length;
        }
        
        // This comment should explain why we're returning 1, since that isn't obvious in the line below
        // where we have || 1
        return 1
```